package.json

```diff
@@ -56,18 +55,19 @@
     "@ethereumjs/config-prettier": "^1.1.1",
     "@ethereumjs/config-tsc": "^1.1.1",
     "@ethereumjs/config-tslint": "^1.1.1",
+    "@types/lru-cache": "^5.1.0",
```

Comment:
I found I had to add this type because `ethereumjs-blockchain` was failing to build in the browser. This might be good motivation to introduce a karma test runner for that repo to catch this and future issues.

The error encountered in karma was:
```
node_modules/ethereumjs-blockchain/dist/cache.d.ts:2:22 - error TS7016: Could not find a declaration file for module 'lru-cache'.

'/Users/rg/dev/ethereumjs-block/node_modules/ethereumjs-blockchain/node_modules/lru-cache/index.js' implicitly has an 'any' type.

  Try `npm install @types/lru-cache` if it exists or add a new declaration (.d.ts) file containing `declare module 'lru-cache';`

2 import * as LRU from 'lru-cache';
```

Reply:
Yeah, I think we should have karma tests along all (actively developed) libraries.